nav2_amcl/include/nav2_amcl/amcl_node.hpp

```diff
@@ -243,7 +243,8 @@ class AmclNode : public nav2_util::LifecycleNode
   tf2::Duration save_pose_period_;
   double sigma_hit_;
   bool tf_broadcast_;
-  tf2::Duration transform_tolerance_;
+  tf2::Duration transform_tolerance_; // tf timeout duration
+  tf2::Duration transform_publish_shift_; // post-date the transform that is published, this was formerly called transform_tolerance_
```

Comment:
Please remove this and just use transform_tolerance_ for all in AMCL

Reply:
Coming in the next commit